plugins/modules/aci_fabric_pod_selector.py

```diff
@@ -66,7 +66,7 @@
 - cisco.aci.owner
 
 notes:
-- The C(pod_profile) must exist before using this module in your playbook.
+- The M(cisco.aci.aci_fabric_pod_profile) must exist before using this module in your playbook.
```

Comment:
```suggestion
- The C(pod_profile) must exist before using this module in your playbook.
- The M(cisco.aci.aci_fabric_pod_profile) module can be used to create the C(pod_profile).
```

I think this might be a better way to explain it and more in line with other modules.